earthfile2llb/parser/EarthParser.g4

```diff
@@ -4,7 +4,7 @@ options {
 	tokenVocab = EarthLexer;
 }
 
-earthFile: NL* stmts? NL+ targets? NL* EOF;
+earthFile: NL* stmts? NL* targets? NL* EOF;
```

Comment:
I think we still need to force a NL if base `stmts` are present.

```suggestion
earthFile: NL* (stmts NL)? NL* targets? NL* EOF;
```

Reply:
This is to enforce a new line between `+base` and `+my-target` right? Just checking for my own understanding

```
FROM xyz
RUN foo

my-target:
    RUN do-it.sh
```

Reply:
not quite.. it's to prevent this (no new line at all):

```
FROM xyz
RUN foo my-target:
    RUN do-it.sh
```

Reply:
Hm I tried that with the old way and it still failed. Willing to go along with it though